images/tier1/imprsonate.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
Weird... there are both `impersonate` and `imprsonate` icons, and they're different... Once again, I'll assume that's intentional on the part of the design team!